modules/features/podcasts/src/main/java/au/com/shiftyjelly/pocketcasts/podcasts/view/ProfileManageDownloadsCard.kt

```diff
@@ -52,36 +56,47 @@ fun ManageDownloadsCard(
             .padding(horizontal = 16.dp).padding(top = 16.dp),
 
     ) {
-        Icon(
-            painter = painterResource(id = IR.drawable.pencil_cleanup),
-            contentDescription = stringResource(LR.string.pencil_clean_up_icon_content_description),
-            modifier = Modifier
-                .padding(end = 12.dp)
-                .size(24.dp),
-            // The icon isn't clickable so the design matches the title color
-            tint = MaterialTheme.theme.colors.primaryText01,
-        )
-
-        Column(
-            modifier = Modifier.padding(end = 16.dp),
-        ) {
-            TextH40(
-                text = stringResource(LR.string.need_to_free_up_space),
-                fontWeight = FontWeight.Bold,
-                color = MaterialTheme.theme.colors.primaryText01,
+        Row(modifier = Modifier.weight(1f)) {
+            Icon(
+                painter = painterResource(id = IR.drawable.pencil_cleanup),
+                contentDescription = stringResource(LR.string.pencil_clean_up_icon_content_description),
+                modifier = Modifier
+                    .padding(end = 12.dp)
+                    .size(24.dp),
+                // The icon isn't clickable so the design matches the title color
+                tint = MaterialTheme.theme.colors.primaryText01,
             )
 
-            Spacer(modifier = Modifier.height(2.dp))
+            Column {
+                TextH40(
+                    text = stringResource(LR.string.need_to_free_up_space),
+                    fontWeight = FontWeight.Bold,
+                    color = MaterialTheme.theme.colors.primaryText01,
+                )
 
-            TextP50(
-                text = stringResource(LR.string.save_space_by_managing_downloaded_episodes, formattedTotalDownloadSize),
-                color = MaterialTheme.theme.colors.primaryText02,
-            )
+                Spacer(modifier = Modifier.height(2.dp))
+
+                TextP50(
+                    text = stringResource(LR.string.save_space_by_managing_downloaded_episodes, formattedTotalDownloadSize),
+                    color = MaterialTheme.theme.colors.primaryText02,
+                )
 
-            TextButton(onClick = { onManageDownloadsClick.invoke() }, contentPadding = PaddingValues()) {
-                TextH50(text = stringResource(LR.string.manage_downloads), color = MaterialTheme.theme.colors.primaryIcon01)
+                TextButton(onClick = { onManageDownloadsClick.invoke() }, contentPadding = PaddingValues()) {
+                    TextH50(text = stringResource(LR.string.manage_downloads), color = MaterialTheme.theme.colors.primaryIcon01)
+                }
             }
         }
+
+        IconButton(
+            onClick = { onMoreOptionsClick.invoke() },
+            modifier = Modifier.size(24.dp),
+        ) {
+            Icon(
+                imageVector = Icons.Default.MoreVert,
```

Comment:
In the designs the 3 dots are in horizontal, but I i think we should display in vertical like in other places. I asked Dom about this p1734608218301639/1734454637.773259-slack-C07TL9YK9V1

Reply:
Dom confirmed that it looks good p1734628669674889/1734454637.773259-slack-C07TL9YK9V1